Site editor: Updated editor styles to match the post editor

MichaelArestad opened the pull request:
Fixes #23183 and a padding/spacing issue with the top bar inserter (and other buttons). Everything should be pretty close, now (though I do think they should share styles rather than repeat them).

#### Before

![image](https://user-images.githubusercontent.com/1123119/85791772-a2069200-b6ef-11ea-9c19-aa3a1b3c7ee3.png)


#### After

![image](https://user-images.githubusercontent.com/1123119/85791628-653a9b00-b6ef-11ea-89e4-f26d060bf1e8.png)

MichaelArestad committed:
Updated editor styles to match the post editor

MichaelArestad committed:
Updated editor styles to match the post editor

MichaelArestad committed:
Remove commented out button styles

MichaelArestad committed:
Remove commented out button styles

youknowriad commented on packages/edit-site/src/style.scss:
```diff
@@ -56,19 +56,6 @@ body.toplevel_page_gutenberg-edit-site {
-	// The button element easily inherits styles that are meant for the editor style.
-	// These rules enhance the specificity to reduce that inheritance.
-	// This is duplicated in visual-editor.
-	& .components-button {
```
These styles are actually needed, they're  just being applied  on the wrong selector which i'm fixing here https://github.com/WordPress/gutenberg/pull/23464

MichaelArestad replied:
@youknowriad Ah! I see. What would you recommend I do with this PR?

youknowriad replied:
Maybe keep this  PR about the inserter toggle?

MichaelArestad replied:
@youknowriad The button styles were part of that. They bled into all sorts of things. I can revert the changes to `edit-site/src/style.scss` to avoid conflicts. Would that work?

youknowriad replied:
yes, so I merged the other branch, you can now rebase this and adapt it.

mkaz committed:
Fix environment misspells (#23489)

epiqueras committed:
Components: Add a WAI-ARIA compliant Combobox. (#19657)

* Components: Implement a combobox control.

* Components: Update Downshift.

* Components: Update Downshift.

ZebulanStanphill committed:
Optimize useSelect calls. (#23255)

talldan committed:
Extract block draggable scroll behaviour into a React hook (#23444)

* Extract block draggable scroll behaviour into React hook

* Use BlockNodes context to get block element

* Update JSDoc

* Check for block nodes being undefined

talldan committed:
Remove REST_WP_REST_Block_Types_Controller_Test which now exists in WordPress core (#23500)

noisysocks committed:
Minor improvements to the release tool and release documentation (#23441)

* Release tool: Make it clearer that you're supposed to insert a GitHub token

* Release tool: Perform a shallow clone on the plugin repository

Specifying `--depth=1` speeds the clone up significantly on slow
connections, and `--no-single-branch` ensures that we can still checkout
non-main branches including remote branches.

* Release documentation: Clarify instrucitons for creating a RC 2

When there are merge conflicts between the `release/x.x` and `master`
branch, things aren't so simple. You have to create a `bump/x.x` branch
so that you don't rebase `release/x.x`.

* Release documentation: Mention `bin/plugin/cli.js perf`

* Release documentation: Use relative /docs URL

diegohaz committed:
Show a warning when SlotFillProvider is missing (#23493)

* Show a warning when SlotFillProvider is missing

* Move warning to the default registerSlot function

ocean90 committed:
Scripts: Remove default exlcude rule for node_modules for SVG, CSS and Sass files (#23495)

youknowriad committed:
Fix Site builder inspector panels buttons (#23464)

mchowning committed:
Run pod install in react-native-editor/ios/ (#23405)

Co-authored-by: Paul Von Schrottky <paul.von.schrottky@automattic.com>

gziolo committed:
Tools: Always append prerelease to version update in package.json when preparing npm relase (#23480)

juanfra committed:
Add `max-width` for the post publish panel buttons. (#23487)

talldan committed:
Try removing some code (#23504)

marecar3 committed:
[RN-MOBILE] fix line height crash (#23507)

* Fix line-height crash

* format fix

* Add block example with lineHeight attribute

youknowriad committed:
Improve the performance of the keyboard shortcuts binding (#23394)

ceyhun committed:
[RNMobile] Fix random React Native E2E Tests failures with errno 134 (#23452)

* Print system information

* Add debugger to iOS workflow

* Remove non-working npm cache clean command

* Remove android runner

* Install watchman

* Bundle iOS 25 times

* Bundle iOS 25 times w/o watchman

* Bundle iOS 25 times with watchman

* Try one last time without watchman

* Remove maxWorkers parameter from metro config

* Remove metro DependencyGraph.js patch

This was introduced in https://github.com/wordpress-mobile/gutenberg-mobile/pull/873
as hack to patch metro to not watch files when bundling. Current 0.58.0 version of
metro that we are using now automatically checks if it's running on CI via 'ci-info'
npm package and doesn't enable watch.

See: https://www.npmjs.com/package/ci-info
See: https://github.com/facebook/metro/blob/v0.58.0/packages/metro/src/node-haste/DependencyGraph.js#L96

* Revert "Remove android runner"

This reverts commit 51ba723841223b1aced43ac04667f086df68bfb0.

* Install watchman on both iOS and Android runners

* Remove script bundling iOS multiple times

* Run React Native E2E tests on pull request instead of push

ellatrix committed:
Drag and drop: enable for multi selection (#23477)

* Fix move to different parent

* Fix unit tests

* Additional tests

* Hide blocks that are being dragged

Commit:
RangeControl: Integrate NumberControl + update internal state flow (#23006)

* Improve roundFloat precision. Integrate NumberControl into RangeControl

* Refactor forwardRef implementation for RangeControl.

Enhance RangeInput with track and rail props + style rendering.
Improve inline docs for rail/utils.js

* Improve drag interaction for NumberControl in RangeControl

* Fix typo for RangeControl ForwardedComponent export

* Add jumpStep feature to range slider. Refactor value parsing and fix tests

* Improve stability of useControlledState. Update integration of useControlledState.

* Consoliate input jumpstep for RangeContorl and NumberControl with new useJumpStep hook

* Remove key UP/DOWN + SHIFT tests for NumberControl and InputControl

The reason is because we're now relying on the native HTML `input[type="number"]` incrementing/decrementing function.
Jump stepping values (holding shift to jump by a multiplier, default of 10) is now being handled by the new useJumpStep hook. This hook listens to `Shift` keyboard presses and modifies the `step` prop that is passed into the `input[type="number"]` element.

* Add tests for useJumpStep hook

* Adjust JSDocs for utils/hooks and utils/math

* Restore UP/DOWN keyboard events for NumberControl (for non type="number" handling)

* Revert "Remove key UP/DOWN + SHIFT tests for NumberControl and InputControl"

This reverts commit 0cc791ccc70029e45ce5ea79079603faba7c033a.

* Fix jumpStep multiplier in NumberControl

* Fix width responsiveness of RangeControl track vs. NumberControl

* Adjust import comment spacing. Add JSDoc comments for space() util.

* Add trackColor and railColor to RangeControl README

* Add JSDocs for getPrecision function

* Fix RangeControl selector for E2E test

* Update switch logic for ns/ew cursor

* Simplify getDefinedValue utility

* Update packages/components/src/utils/hooks/use-controlled-state.js

Co-authored-by: Andrés <nosolosw@users.noreply.github.com>

* Move className for InputRange to root index.js

* Update shiftStep in number-control README

* Add isShiftStepEnabled in RangeControl README

* Add note on NumberControl in CHANGELOG

Co-authored-by: Andrés <nosolosw@users.noreply.github.com>

Commit:
Gallery: Update button rendering (Move backward, Move forward, Remove) (#23471)

* Gallery: Update button rendering (Move backward, Move forward, Remove)

* Replace close icon with closeSmall

youknowriad committed:
Add missing notices dependency (#23517)

youknowriad committed:
Move the JS unit tests from travis to github actions (#23518)

youknowriad committed:
Move the php unit tests to Github actions (#23520)

youknowriad committed:
Move the end2end tests from Travis to Github actions (#23523)

MichaelArestad committed:
Merge branch 'fix/site-editor-style-bugs' of https://github.com/WordPress/gutenberg into fix/site-editor-style-bugs

MichaelArestad commented:
Messed up a rebase. Just spinning up a new PR.
